app/models/assignment.rb

```diff
@@ -126,7 +127,7 @@ def run_tests!(params)
   end
 
   def to_resource_h
-    as_json(except: [:exercise_id, :submission_id, :id, :submitter_id, :solution, :created_at, :updated_at, :submission_status],
+    as_json(except: [:exercise_id, :submission_id, :organization_id, :id, :submitter_id, :solution, :created_at, :updated_at, :submission_status],
```

Comment:
I think we should expose the organization name in this resource hash

Reply:
The organization name is exposed on line 141, isn't it?

Reply:
You are right. Dismiss my comment